src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -642,7 +627,7 @@ public void testMultiThread() throws Exception {
         .execute(
             String.format("create or replace table %s (numcol NUMBER(10,2));", multiThreadTable));
     int numThreads = 20;
-    int numRows = 1000000;
+    int numRows = 10000;
```

Comment:
any issue with the old value?

Reply:
It was excessive and caused the tests to run longer than necessary